content/guides/getting-started/rust.md

```diff
@@ -3,9 +3,434 @@ title: "Run a rust-libp2p node"
 weight: 4
 description: "Learn how to run a rust-libp2p node and use the ping protocol"
 aliases:
-    - "/tutorials/rust"
-    - "/guides"
-    - "/guides/rust"
+- "/tutorials/rust"
+- "/guides"
+- "/guides/rust"
 ---
 
 Check out [tutorials of the Rust libp2p implementation](https://docs.rs/libp2p/newest/libp2p/tutorials/index.html).
+
+This is the first in a series of tutorials on working with libp2p's Rust implementation,
+[js-libp2p](https://github.com/libp2p/rust-libp2p).
```

Comment:
```suggestion
[rust-libp2p](https://github.com/libp2p/rust-libp2p).
```

Reply:
It's easier to find via search engine here. I had to dig to find the version you're referring to, which would also need to be updated for
https://github.com/libp2p/rust-libp2p/pull/3509
I added a number of comments to make things clearer, as does the verbosity in repeating some code just to be explicit. The prettier format of having code blocks and empty space is also more condusive to those who have dyslexia.

Reply:
I appreciate your work here.

The benefit of the tutorial in the rust-libp2p repository is, that it is never going out-of-date. In my eyes this outweights your above arguments in favor of this pull request.

Mind proposing your changes on rust-libp2p instead and making it better discoverable here instead of copying it?